server/app/mutations/stacks/create.rb

```diff
@@ -57,6 +61,20 @@ def execute
       stack
     end
 
+    # @param [Array<Hash>] volumes
+    def create_volumes(volumes)
+      return unless volumes
+      volumes.each do |volume|
+        unless volume[:external]
+          outcome = Volumes::Create.run(grid: self.grid, **volume.symbolize_keys)
```

Comment:
AFAIK this prevents installing multiple copies of the same stack, and if fix this naming later, it will lose any installed stack volumes.